app/service/update-appeal-service.ts

```diff
@@ -426,6 +426,10 @@ export default class UpdateAppealService {
       reheardRule35Document = this.mapSupportingDocumentToEvidence(caseData.ftpaR35AppellantDocument, documentMap);
     }
 
+    if (caseData.ftpaR35RespondentDocument && caseData.ftpaR35RespondentDocument.document_filename) {
```

Comment:
Do you need the DLRM flah check here or it is enough with the caseData check?

Reply:
caseData check is enough in this scenario, will only have data when toggle is on.